Updated documentation

johnbaldwin opened the pull request:
- Added a FAQ file
- Added a HISTORY file
- Added a PyPI specific README for the PyPI package description
- Updated the project setup.py to use the new PyPI specific README


You can see the PyPI specific README in the project description: https://test.pypi.org/project/figures/

OmarIthawi commented on HISTORY.md:
```diff
@@ -0,0 +1,18 @@
+RELEASE HISTORY
```
`HISTORY.md` -> `CHANGELOG.md`? Not important at all though.

johnbaldwin replied:
I've been on this fence on this myself. I think I spent a couple of hours poring over how other repos did the changelog and faq docs. `CHANGELOG` *would* be easier to file search for in GitHub global search

OmarIthawi commented on setup.py:
```diff
@@ -17,7 +17,7 @@
 from setuptools import find_packages, setup
 
-with open(os.path.join(os.path.dirname(__file__), 'README.rst')) as readme:
+with open(os.path.join(os.path.dirname(__file__), 'docs/readme-pypi.rst')) as readme:
```
I can't find this file. Where am I supposed to look?

I did:
```
$ git checkout develop
$ git pull
$ git grep pypi
```

Nothing relevant showed up.

johnbaldwin replied:
@OmarIthawi Dang! I forgot to add this file when I had to redo the branch. Added!

johnbaldwin commented:
@OmarIthawi I'm also working up some more info in the main README to make it clear that it runs in Ginkgo but not yet other named releases. Also I'm adding instructions for submitting bugs

johnbaldwin commented:
@OmarIthawi I renamed 'HISTORY' to 'CHANGELOG', added the missing PyPI readme and updated the main readme. The build failure is from the intermittent test bug, which I'm going to have to address soon as this is getting old

johnbaldwin commented:
I just removed a docstring from one of the views. It wasn't helpful.

OmarIthawi commented on CHANGELOG.md:
```diff
@@ -0,0 +1,18 @@
+RELEASE HISTORY
```
```suggestion
CHANGELOG
```

Or, `Release History`

It just feels like someone's shouting.

johnbaldwin replied:
Can do

johnbaldwin replied:
Changed to 'Release History' since that is what it really is, w/out shouting
